packages/rs-drive-proof-verifier/src/types.rs

```diff
@@ -12,8 +12,19 @@ use dpp::{
     document::Document,
     identity::KeyID,
     prelude::{DataContract, Identifier, IdentityPublicKey, Revision},
+    util::deserializer::ProtocolVersion,
 };
 
+/// Collection of objects returned by the [FetchMany](crate::platform::FetchMany) operation.
+///
+/// Collection of objects of type `O`, indexed by key `K`.
+///
+/// Object is an option, representing:
+///
+/// * `Some(O)` - object is found
+/// * `None` - object is not found, and the platform provided proof of non-existence
+pub type Collection<K, O> = BTreeMap<K, Option<O>>;
```

Comment:
Not sure Collection is the right word here.

Reply:
renamed to RetrievedObjects